### ISSUE TITLE
openapi-to-graphql fails when definitions contains enum

### ISSUE THREAD

**Issue by folding** [IBM/openapi-to-graphql#217]
I get this error message:

`OpenAPI-to-GraphQL creation event error:  str.split is not a function`

When I run this command:

`openapi-to-graphql v1-split.json --save split.graphql`

Here are the contents of v1-split.json:

```
{
  "swagger": "2.0",
  "info": {
    "version": "v1",
    "title": "Test.API"
  },
  "host": "localhost:5234",
  "schemes": [
    "http"
  ],
  "paths": {
    "/api/v1/email/send": {
      "post": {
        "tags": [
          "Email"
        ],
        "summary": "Sends the given email via SendGrid",
        "description": "Sends the given email via SendGrid",
        "operationId": "Email_SendEmailAsync",
        "consumes": [
        ],
        "produces": [
          "application/json"
        ],
        "parameters": [
          {
            "name": "emailData",
            "in": "body",
            "description": "",
            "required": true,
            "schema": {
              "$ref": "#/definitions/SendEmailData"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "OK",
            "schema": {
              "$ref": "#/definitions/SendEmailResult"
            }
          }
        }
      }
    }
    },
  "definitions": {
    "SendEmailData": {
      "type": "object",
      "properties": {
        "EmailQueueId": {
          "format": "int32",
          "type": "integer"
        },
        "FromAddress": {
          "type": "string"
        },
        "FromName": {
          "type": "string"
        },
        "ReplyTo": {
          "type": "string"
        },
        "ToAddress": {
          "type": "string"
        },
        "Subject": {
          "type": "string"
        },
        "PlainTextContent": {
          "type": "string"
        },
        "HtmlContent": {
          "type": "string"
        }
      }
    },
    "SendEmailResult": {
      "type": "object",
      "properties": {
        "StatusCode": {
          "format": "int32",
          "enum": [
            100,
            101,
            200,
            201,
            202,
            203,
            204,
            205,
            206,
            300,
            300,
            301,
            301,
            302,
            302,
            303,
            303,
            304,
            305,
            306,
            307,
            307,
            400,
            401,
            402,
            403,
            404,
            405,
            406,
            407,
            408,
            409,
            410,
            411,
            412,
            413,
            414,
            415,
            416,
            417,
            426,
            500,
            501,
            502,
            503,
            504,
            505
          ],
          "type": "integer"
        },
        "ErrorJson": {
          "type": "string"
        }
      }
    }
    },
  "securityDefinitions": {
    "oauth2": {
      "type": "oauth2",
      "description": "OAuth2 Implicit Grant",
      "flow": "implicit",
      "authorizationUrl": "http://localhost:5897/core/connect/authorize",
      "tokenUrl": "http://localhost:5297/core/connect/token",
      "scopes": {
        "swagger": "Swagger UI"
      }
    }
  }
}
```

The operation succeeds when I remove the enum from the definition.  Let me know if you need any other info.

**Comment by Alan-Cha** [comment]
@folding As a first thought, GraphQL requires enum values to be of type string and OtG is not properly ensuring that the values are converted, if necessary.

**Comment by folding** [comment]
@Alan-Cha, adding quotes around each enum did allow it to generate successfully.   Like so:

```
...
"SendEmailResult": {
      "type": "object",
      "properties": {
        "StatusCode": {
          "format": "int32",
          "enum": [
            "100",
            "101",
            "200",
            "201",
            "202",
...
```

**Comment by Alan-Cha** [comment]
@folding Thanks for checking! I will try to make the necessary changes sometime soon, although feel free to send us a PR if you want to take a shot at it yourself. In any case, thank you for filling this issue! Very helpful!

**Comment by folding** [comment]
@Alan-Cha I'll poke at it a little this weekend and try to narrow down where to put a fix

**Comment by folding** [comment]
@Alan-Cha I have a fix in my fork but it messed with a bunch of formatting in example_api.test.ts.  I didn't see the changes in until after the commit... the tests are all passing

**Comment by folding** [comment]
After much squashing on my branch I have a clean pull request submitted.

**Comment by Alan-Cha** [comment]
@folding Oh wow! I did not expect such a fast turnaround time! I was going to take a look at the code base to give you some pointers on where to look but you already submitted a PR! Thank you so much!!!

**Comment by Alan-Cha** [comment]
Sorry for the delay! Fixed after accepting PR #218
